docs/reference/field-types/relationship-reverse.md

```diff
@@ -1,36 +1,40 @@
 # `relationshipReverse`
 
-<!-- TODO: This whole page needs to be revisited following successful testing of the `relationshipReverse` field type. -->
-A `relationshipReverse` field allows us to access the "other side" of a doc [relationship](/reference/field-types/relationship.md). There is no editing interface for this field type as the relationship is defined in the paired `relationship` field. `relationshipReverse` fields are a convenience allowing developers to identify Apostrophe documents that have established relationships with a given document.
+Adding a `relationshipReverse` field to a doc type schema reveals Apostrophe docs that have established relationships with a given doc. It is a developer convenience, showing the "other side" of a [`relationship` field](/reference/field-types/relationship.md). There is no editing interface for this field type as the relationship is defined in the paired `relationship` field.
+
+Take the example of a website that has a `pizza` piece type with a `relationship` field connecting to its `topping` piece type. Each pizza piece chooses the registered toppings that it has (the relationship is _pizza-to-topping_). The website might also want to list all available toppings on a page and show all pizzas that use each topping (the _topping-to-pizza_ direction). A `relationshipReverse` field could make it easy to find and display that information.
```

Comment:
**"It is a developer convenience, showing the "other side" of a [`relationship` field](/reference/field-types/relationship.md).**" I think we could remove this bit, or at least rephrase..**.."the other side"** feels colloquial and not entirely useful.


The website might also want to list all available toppings on a page and show all pizzas that use each topping (the _topping-to-pizza_ direction). **A `relationshipReverse` field could make it easy to find and display that information.** I think this can be removed too for brevity.

pizza piece is lol.

Reply:
I'm not sure what you're saying for the second paragraph. Remove the whole paragraph? The sentence you bolded is the whole point of that paragraph.

Reply:
...
...
...

I am happy with what I currently see in this PR re: the above conversation. A little confused as to the back and forth, but maybe I don't need to get into that since I'm happy with what is there as of today.

Reply:
I still think the last sentence could be removed.

"Each pizza piece chooses the registered toppings that it has" -- chooses here might be a bit strange.